include/range/v3/detail/optional.hpp

```diff
@@ -37,7 +37,7 @@ namespace ranges
             optional(T t)
               : data_(emplaced_index<1>, std::move(t))
             {}
-            template <typename...Args, CONCEPT_REQUIRES_(Constructible<T, Args &&...>())>
+            template <typename...Args, CONCEPT_REQUIRES_(Constructible<T, Args...>())>
```

Comment:
Not this time ;) `Constructible` have been cured of that madness and use `std::declval`. I REALLY need to find some time to transition the rest of the library.